contracts/interfaces/IERC173.sol

```diff
@@ -5,12 +5,6 @@ pragma solidity ^0.8.0;
 ///  Note: the ERC-165 identifier for this interface is 0x7f5828d0
 /* is ERC165 */
 interface IERC173 {
-    /// @dev This emits when ownership of a contract changes.
-    event OwnershipTransferred(
```

Comment:
This event isn't needed?

Reply:
@cinnabarhorse 
Yes, this function is not needed for this interface. i checked other diamonds.